docs/source/getting-started/content/proc_starting-monitoring-alerting-telemetry.adoc

```diff
@@ -0,0 +1,16 @@
+[id="starting-monitoring-alerting-telemetry_{context}"]
+= Starting Monitoring, Alerting, and Telemetry
+
+In order to make sure {prod} can run on a typical laptop, some resource–heavy services get disabled by default.
+One of these is Prometheus and all the related monitoring, alerting, and telemetry functionality, with the latter being
+responsible for your cluster being listed in the link:https://cloud.redhat.com/openshift[Red Hat OpenShift Cluster
+Manager].
+
+.Procedure
```

Comment:
A running cluster with access to `oc` should probably be listed as a prerequisite for this procedure. :)

We have access to the cluster via `oc` documented in the "Accessing the OpenShift cluster" section of the docs. FYI, we can `xref` this with `<<accessing-the-openshift-cluster_{context}>>`.